cardano-cli/src/Cardano/CLI/EraBased/Options/Genesis.hs

```diff
@@ -198,6 +198,15 @@ pGenesisCreateStaked envCli =
     <*> pBulkPoolsPerFile
     <*> pStuffedUtxoCount
     <*> Opt.optional pRelayJsonFp
+  where
+    pRelayJsonFp :: Parser FilePath
+    pRelayJsonFp =
+      Opt.strOption $ mconcat
+        [ Opt.long "relay-specification-file"
+        , Opt.metavar "FILE"
+        , Opt.help "JSON file specified the relays of each stake pool."
```

Comment:
"that specifies"

Reply:
Since this is in `create-staked`, I chose not to change anything :slightly_smiling_face: